src/generic_core/remote-connection.ts

```diff
@@ -23,6 +23,17 @@ module Core {
 
     private isUpdatePending_ = false;
 
+    // Resolve this promise when rtcToNet is created and therefore not null.
+    // Used to help determine when to call handleSignal (which relies
+    // on rtcToNet or socksToRtc being not null).
+    // The promise is reset in resetRtcToNetCreated().
+    public rtcToNetCreated :Promise<void> = new Promise<void>((F, R) => {
```

Comment:
Can we change the name to "onceRtcToNetCreated"?  That way it matches the convention we use elsewhere in the code and uproxy-networking